webflow-hacks/misc-hacks/51/51.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+document.addEventListener('DOMContentLoaded', () => {
+  const TABLE_SELECTOR = '[fs-hacks-element="table"]';
+  const COLUMN_SELECTORS = [
+    {
+      type: '[fs-hacks-element="design-column"]',
+      total: '[fs-hacks-element="design-total-fee"]',
+    },
+    {
+      type: '[fs-hacks-element="development-column"]',
+      total: '[fs-hacks-element="development-total-fee"]',
+    },
+    {
+      type: '[fs-hacks-element="subscription-column"]',
+      total: '[fs-hacks-element="subscription-total-fee"]',
+    },
+  ];
+  // loop through all tables
+  const tables = document.querySelectorAll(TABLE_SELECTOR);
+  tables.forEach((table) => {
+    updateSubTotals(table, COLUMN_SELECTORS);
+  });
```

Comment:
Feels unnecessary to pass the `COLUMN_SELECTORS` variable as a parameter in the `updateSubtotals()` function. This is an immutable constant, not a dynamic value, and could be perfectly placed outside the `DOMContentLoaded` so any function can access it.